components/server/src/oauth-server/db.ts

```diff
@@ -57,12 +57,10 @@ const jetBrainsGateway: OAuthClient = {
     ],
 };
 
-function createVSCodeClient(protocol: "vscode" | "vscode-insiders" | "vscodium"): OAuthClient {
+function createVSCodeClient(protocol: string, displayName: string): OAuthClient {
```

Comment:
What's the reason we had the restriction on protocol before? Do/Can we (need to) keep it?

Reply:
@geropl we used to only support VS Code and VS Code Insiders (with a later addition of VSCodium) because of constraints of the VS Code Marketplace. Now that we will publish the extension to Open VSX as well, we can broaden the IDE list to anything which supports the VS Code Extensions API.